SwiftTweaks/TweakWindow.swift

```diff
@@ -74,25 +74,46 @@ import UIKit
 
 		super.init(frame: frame)
 
-		tintColor = AppTheme.Colors.controlTinted
-
-		if tweakStore.enabled {
-			switch gestureType {
-			case .gesture(let gestureRecognizer):
-				gestureRecognizer.addTarget(self, action: #selector(self.presentTweaks))
-			case .twoFingerDoubleTap:
-				let twoFingerDoubleTapGesture = UITapGestureRecognizer()
-				twoFingerDoubleTapGesture.numberOfTapsRequired = 2
-				twoFingerDoubleTapGesture.numberOfTouchesRequired = 2
-				twoFingerDoubleTapGesture.addTarget(self, action: #selector(self.presentTweaks))
-				self.addGestureRecognizer(twoFingerDoubleTapGesture)
-			case .shake:
-				break
-			}
-		}
+		commonInit(tweakStore: tweakStore)
```

Comment:
Thanks for moving this to a commonInit rather than having lots of duplicated code!